client-course-schedulizer/src/utilities/services/addSectionService.ts

```diff
@@ -142,20 +203,23 @@ export const mapInputToInternalTypes = (data: SectionInput) => {
         days: data.days,
         duration: Number(data.duration),
         location: {
-          building: location[0],
+          building,
           roomCapacity: Number(data.roomCapacity),
-          roomNumber: location[1],
+          roomNumber,
         },
         startTime: startTimeCase(data.startTime),
       },
     ],
     semesterLength: semesterType,
     studentHours: Number(data.studentHours),
     term: data.term,
-    year: "2021-2022",
+    // TODO: there currently isn't a way to get the year for this Section.
+    year: `${"hello"}`,
```

Comment:
For testing with dept chairs soon, should we have it assume 2021? (Also I noticed in the `Course Section Enrollment.csv` that they list 2020 as the academic year for all classes, even if the class is in spring when it's 2021)

Reply:
I like your fix in pr #69 . Thanks